hack/azure-cluster-up.sh

```diff
@@ -63,8 +72,12 @@ echo -e "\nSpinning up Azure Kubernetes Cluster $CLUSTER_NAME"
 az aks create --resource-group $RESOURCE_GROUP --name $CLUSTER_NAME --enable-addons monitoring --generate-ssh-keys --service-principal $username --client-secret $password
 
 echo -e "\nGetting Credentials for Cluster $CLUSTER_NAME"
-az aks get-credentials --resource-group $RESOURCE_GROUP --name $CLUSTER_NAME
+az aks get-credentials --resource-group $RESOURCE_GROUP --name $CLUSTER_NAME --overwrite-existing
 echo -e "\n"
 
-DRIVER_NAME=$(dirname "$(realpath ${BASH_SOURCE[0]})")
-source "$DRIVER_NAME/cosi-install.sh"
\ No newline at end of file
+if [ $VERSION = "local" ] || [ $VERSION = "push" ];
```

Comment:
what would be the else for this statement?

Reply:
The default will just be remote. I hadn't yet implemented an else, since we don't really have any versions/modes other than the master yet.

Reply:
I understand, but if someone were to pass a version, this set of code wouldn't be triggered. Let's fix this part and complete the PR.

Reply:
Implemented some versioning, but that code is intentionally not triggered, using curly braces to call curl commands essentially takes both scripts and lines them up as a single script to run it. calling the cosi-install.sh script is unnecessary and will probably cause an error